eng/pipelines/runtime.yml

```diff
@@ -131,832 +131,43 @@ jobs:
       - eng/pipelines/libraries/*
 
 #
```

Comment:
Please don't forget to undo the leg deletions in runtime.yml before merging in.

Reply:
Yeah. I am waiting for it to complete a run, hopefully giving me all the test failures. Then I will exclude those, and put all the other legs back.